src/main/java/org/jabref/gui/groups/GroupDialogHeader.java

```diff
@@ -0,0 +1,5 @@
+package org.jabref.gui.groups;
+
+public enum GroupDialogHeader {
```

Comment:
Maybe its just a question of personal preference, but I would suggest to inline this simple enum in GroupDialogView instead of a new file, since the scope of this is extremely limited. Do we have a policy about this @JabRef/developers ?

Reply:
Hi @calixtus, since this enum is used in multiple files inside the package, thus the new file is crated. And from some reading i did, it's best practise for enum to be declared in a file of its own if it's public, but again some clarifications from the devs would be much appreciated.

Reply:
I think both work. Since it is only used in connection with the group edit dialog, one could argue that this is the right place. But a new file is also ok in my opinion.